torch/nn/functional.py

```diff
@@ -3895,7 +3895,11 @@ def interpolate(input: Tensor, size: Optional[int] = None, scale_factor: Optiona
                     f"input with spatial dimensions of {list(input.shape[2:])} and output size of {size}. "
                     "Please provide input tensor in (N, C, d1, d2, ...,dK) format and "
                     "output size in (o1, o2, ...,oK) format."
-
+                )
+            if not all(isinstance(x, (int, torch.SymInt)) for x in size):
```

Comment:
```suggestion
            if not torch.jit.is_scripting() and not all(isinstance(x, (int, torch.SymInt)) for x in size)):
```

I never know what is wrong with jit... My usual trick is just to disable it like this.